cmd/validator/validator.go

```diff
@@ -148,7 +153,7 @@ func mainInit() int {
 	// since the exclude dirs are a comma separated string
 	// it needs to be split into a slice of strings
 	excludeDirs := strings.Split(*validatorConfig.excludeDirs, ",")
-	reporter := getReporter(validatorConfig.reportType)
+	reporter := getReporter(validatorConfig.reportType, validatorConfig.output)
```

Comment:
What is supposed to happen if the output option if specified and the string is empty?  Be sure to add a test for that and set the appropriate response.

Reply:
I added some test cases. Please check them. ae9c479a9e20d6e99ce73ecb9b3a2c61863468d6

Reply:
Looks good. Thanks.